tensorflow/python/ops/signal/shape_ops.py

```diff
@@ -73,16 +73,28 @@ def frame(signal, frame_length, frame_step, pad_end=False, pad_value=0, axis=-1,
   audio = tf.random.normal([3, 9152])
 
   # Compute overlapping frames of length 512 with a step of 180 (frames overlap
-  # by 332 samples). By default, only 50 frames are generated since the last
-  # 152 samples do not form a full frame.
+  # by 332 samples). By default, only 49 frames are generated since a frame
```

Comment:
😱 thank you for this @jmsmdy! This is a reminder to migrate all of tf.signal to [doctests](https://www.tensorflow.org/community/contribute/docs_ref) asap!